src/main/webapp/WEB-INF/static/js/fieldbook-common.js

```diff
@@ -989,45 +989,28 @@ function showBaselineTraitDetailsModal(id) {
 	'use strict';
 
 	if (id !== '') {
-		$.ajax({
-			url: '/Fieldbook/manageSettings/settings/details/' + id,
-			type: 'GET',
-			cache: false,
-			success: function (html) {
-				$('.variable-details-section').empty().append(html);
-				if ($('#selectedStdVarId').length != 0) {
-					$('#selectedStdVarId').val(id);
-				}
-				$('#variableDetailsModal').modal('toggle');
-				if ($('#variableDetailsModal')) {
-					var variableName = $('#ontology-tabs').data('selectedvariablename');
-					$('#variableDetailsModal .modal-title').html(variableDetailsHeader + ' ' + variableName);
-				}
-			}
-		});
-	}
-}
-
-function showBaselineTraitDetailsModal(id, variableTypeId) {
-	'use strict';
 
-	if (id !== '') {
-		$.ajax({
-			url: '/Fieldbook/manageSettings/settings/details/' + variableTypeId + '/' + id,
-			type: 'GET',
-			cache: false,
-			success: function (html) {
-				$('.variable-details-section').empty().append(html);
-				if ($('#selectedStdVarId').length != 0) {
-					$('#selectedStdVarId').val(id);
-				}
-				$('#variableDetailsModal').modal('toggle');
-				if ($('#variableDetailsModal')) {
-					var variableName = $('#ontology-tabs').data('selectedvariablename');
-					$('#variableDetailsModal .modal-title').html(variableDetailsHeader + ' ' + variableName);
-				}
-			}
-		});
+		const crop = $('#cropName').val() ? $('#cropName').val() : cropName;
+		const programUUID = $('#currentProgramId').val() ? $('#currentProgramId').val() : currentProgramId;
```

Comment:
`$('#cropName').val()` look like some legacy way of doing it, perhaps it was set in some hidden input? I don't think it's used anymore, probably the global variable should be enough..

Reply:
I had to do that, because in some pages the variable cropName was undefined.